themes/custom/az_barrio/js/global.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * Global utilities.
+ *
+ */
+(function ($, Drupal) {
+
+  'use strict';
+
+  Drupal.behaviors.bootstrap_barrio = {
```

Comment:
Should we rename this javascript behavior?
```suggestion
  Drupal.behaviors.az_barrio = {
```